setup.py

```diff
@@ -17,15 +17,15 @@
     long_description = readme.read()
 
 validate_dependencies = [
-    "pytest>=6.2.5,<7",
+    "pytest>=6.2.5",
     "pytest-asyncio<1",  # for async
     "Flask-Sockets>=0.2,<1",
     "Flask>=1,<2",  # TODO: Flask-Sockets is not yet compatible with Flask 2.x
     "Werkzeug<2",  # TODO: Flask-Sockets is not yet compatible with Flask 2.x
     "itsdangerous==1.1.0",  # TODO: Flask-Sockets is not yet compatible with Flask 2.x
     "Jinja2==3.0.3",  # https://github.com/pallets/flask/issues/4494
     "pytest-cov>=2,<3",
-    "flake8>=5,<6",
+    "flake8>=5",
```

Comment:
we can use 6.1 instead

Reply:
Unfortunately, `flake8` 6.x requires Python >= 3.8.1, so `flake8` 5.x should be kept in order to stay compatible with Python 3.6/3.7.

I've updated the constraints and added some corresponding comments.

Reply:
Thanks, this makes sense!